src/components/Common/Button.tsx

```diff
@@ -1,9 +1,16 @@
 import classNames from 'classnames';
-import PropTypes from 'prop-types';
+import React from 'react';
 
 import styles from 'components/Common/Button.module.css';
 
-function Button({ className, text, onClick, children }) {
+interface ButtonProps {
```

Comment:
💬 
core한 button 컴포넌트로 보입니다!

how about 
```suggestion
interface ButtonProps extends React.ComponentProps<'button'> {
  text?: string;
}
```

HTML `<button>` 태그가 기본적으로 제공하는 속성들을 컴포넌트에서 직접 사용하고자 하는 경우, 모든 속성을 하나씩 `props`로 추가하는 대신, React의 기본 `<button>` 속성 타입을 확장하는 방법이 효과적입니다. 

이렇게 하면, 추가적인 속성을 선언하지 않아도 HTML button 태그에서 사용할 수 있는 모든 속성을 자동으로 컴포넌트에서 사용할 수 있게 됩니다.

단점은, 불필요한 (사용하지 않는) props들도 자동완성할 때 다 suggestion해준다 입니다.


- [참고 React Typescript Cheatsheet](https://react-typescript-cheatsheet.netlify.app/docs/react-types/componentprops/)